src/style.css

```diff
@@ -0,0 +1,159 @@
+/* root */
+:root {
+  --main-color: rgb(6, 11, 53);
+  --secoundary-color: rgb(97, 96, 112);
+  --distance-to-border: 16px;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+/* reset */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+p,
+h1,
+h2 {
+  margin: 0;
+}
+
+/* card */
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+/* image */
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+
+/* description */
+
+.card__description {
+  margin: 40px var(--distance-to-border) 16px;
```

Comment:
use paddings in `.card` for horizontal indents

try to use `margin-bottom` for vertical indents